analyze.sh

```diff
@@ -14,7 +14,7 @@ if [ -d "$ARTIFACT_DIR" ]; then
   cp public/dist/report.html "${ARTIFACT_DIR}"
 fi
 
-MAX_BYTES=3145728 # ~3 MiB
+MAX_BYTES=3250586 # ~3.1 MiB
```

Comment:
fyi @spadgett

Reply:
When I analyzed the bundles, I noticed that some of the modules were shifted around, and some into the vendors bundle. Overall, the webpack bundle actually decreased in size, but this specific vendors bundle went up a bit.

Reply:
To expand further, the vendor bundle gained around 190 KB, which mostly comes from the addition of the quickstart lib (+93.2 KB), and the showdown package (+93.64 KB) which webpack automatically moved into this bundle (it used to live in a a separate chunk before).
Comparing the whole parsed bundle size at the time of writing, master is at 25.03 MB, and this branch is at 25.02 MB

Reply:
@jschuler Sorry, I missed this comment. If we're going to increase the limit, we need more room. If you set it at just above what we're currently using, minor changes in size in our dependencies will cause failures (as seems to have happened in #9419). This job is meant to catch large dependencies that are accidentally added to the main vendor bundle.

The important bundle is the main vendor bundle as that's what's loaded on console load. The other bundles are lazy-loaded. It's better to have a smaller main vendor bundle even if the total size is a little larger.

Is there a reason we couldn't lazy-load the quick starts library?

If we're pulling showdown into the main vendor bundle, we should remove the code that lazy loads it in other places. But it would be nice to lazy-load libraries of significant size when it makes sense.

I notice also that xterm and victory somehow have been pulled into the main vendor bundle likely in an unrelated change. We should look at these as well.